ouroboros-consensus/src/Ouroboros/Consensus/Mempool/API.hs

```diff
@@ -103,7 +104,7 @@ data Mempool m blk idx = Mempool {
       -- they have already been included. (Distinguishing between these two
       -- cases can be done in theory, but it is expensive unless we have an
       -- index of transaction hashes that have been included on the blockchain.)
-      addTxs :: [(GenTxId blk, GenTx blk)] -> m [(GenTx blk, ApplyTxErr blk)]
+      addTxs :: [(GenTxId blk, GenTx blk)] -> m [(GenTx blk, Maybe (ApplyTxErr blk))]
```

Comment:
Nitpicking: this is the only line in the file that is too long :slightly_smiling_face:. You could wrap it like:
```haskell
addTxs :: [(GenTxId blk, GenTx blk)]
       -> m [(GenTx blk, Maybe (ApplyTxErr blk))]
```

Reply:
Nitpicking is fine by me. I appreciate any comments. Thanks.